test/CMakeLists.txt

```diff
@@ -51,13 +86,15 @@ function(add_performance_test TEST_NAME COMM RMW_IMPLEMENTATION)
       "PERFORMANCE_REPORT_PNG=${PERFORMANCE_REPORT_PNG}"
       "PERFORMANCE_REPORT_CSV=${PERFORMANCE_REPORT_CSV}"
       "RMW_IMPLEMENTATION=${RMW_IMPLEMENTATION}"
+      "RMW_FASTRTPS_USE_QOS_FROM_XML=${RMW_FASTRTPS_USE_QOS_FROM_XML}"
+      "FASTRTPS_DEFAULT_PROFILES_FILE=${FASTRTPS_DEFAULT_PROFILES_FILE}"
```

Comment:
As I explained [here](https://github.com/ros2/buildfarm_perf_tests/pull/26#discussion_r388146433), Fast-RTPS `ros2-eloquent` branch (1.9.3 + bugfixes) does not support synchronous sending of fragments. This means that in that branch, topics `PointCloud512k`, `Array1m`,  and `Array2m` cannot be sent synchronously in ROS 2 Eloquent (either from standalone Fast-RTPS, or from `rmw_fastrtps_*`. I would display some message similar to the ones about the sync mode in different implementations.

However, this issue has been addressed in [1.9.4](https://github.com/eProsima/Fast-RTPS/pull/898) and [master](https://github.com/eProsima/Fast-RTPS/pull/1029) branches, so it will be possible to do it in ROS 2 Foxy.

Reply:
For now this branch is focused on master (Foxy). I will create a branch for each one of the other ROS distros (Dashing or Eloquent) if it is needed

Reply:
That's great to hear!

I just wanted to be sure cause I think [these last jobs](http://3.83.10.11/job/Eci__nightly-performance-overhead-multi_ubuntu_bionic_amd64/52/) are using [this upstream](http://3.83.10.11/job/Eci__nightly-extra-rmw-release_ubuntu_bionic_amd64/104/), which as far as I can tell is setting Fast-RTPS to [this commit](https://github.com/eProsima/Fast-RTPS/commit/75f5155787e949b08bfd97c3a6c39254d0240c31), which is actually (since this morning) not the [head of `ros2-eloquent`](https://github.com/eProsima/Fast-RTPS/commits/ros2-eloquent).

Is there any plan on supporting different ROS 2 version at the same time?

Reply:
The idea is to support all ROS versions which are being build in ci.ros2.org

Reply:
Alright, then I think this thread can be resolved.

As for the different ROS 2 versions, let me know when you start on that and I'll give you hand, cause there are some major differences in Fast-RTPS between Dashing and Eloquent/Foxy (mainly the intra-process delivery)